cmd/kubeadm/app/cmd/util/join.go

```diff
@@ -74,11 +74,16 @@ func GetJoinCommand(kubeConfigFile string, token string, skipTokenPrint bool) (s
 		"Token":          token,
 		"CAPubKeyPins":   publicKeyPins,
 		"MasterHostPort": strings.Replace(clusterConfig.Server, "https://", "", -1),
+		"UploadCerts":    uploadCerts,
+		"CertificateKey": string(key),
```

Comment:
Trying the PR what I see printed is garbage so I cannot properly use this for join:

```
  kubeadm join 172.28.128.55:6443 --token 5ynki1.3erp9i3yo7gqg1nv --discovery-token-ca-cert-hash sha256:c15a8ff2636663672adec5a47492508e8e41772a65fc5e75b82bb11fd5cce127 --certificate-key 
                                                                                                                                                                                           E
Γ�J\���ހ��e��1��Y�Q;
```

Reply:
Sorry, it's on my todo.

Reply:
Ugh, missed it from your PR description, sorry!